roles/hosted_engine_setup/tasks/apply_openscap_profile.yml

```diff
@@ -10,10 +10,10 @@
   stat:
     path: "{{ oscap_datastream }}"
   register: oscap_ds_stat
-- name: Set default OpenSCAP profile
+- name: Set OpenSCAP profile
   shell: >-
     set -euo pipefail && oscap info --profiles {{ oscap_datastream }} |
-    grep -Ei "(standard|disa)" | sort | tail -1 | cut -d':' -f1
+    grep -Ei "({{ he_openscap_profile_name }}:)" | sort | tail -1 | cut -d':' -f1
```

Comment:
Why user cannot simply specify the whole name instead of this filtering?
Also this won't work for disa because there are two disa profiles: 
- xccdf_org.ssgproject.content_profile_stig
- xccdf_org.ssgproject.content_profile_stig_gui

And the command will actually select the gui one, which is not what we want.

Reply:
> Why user cannot simply specify the whole name instead of this filtering?

The user simply needs to specify `stig` or `pci-dss` which IMO is easier than specifying the entire name.
Why do we need this prefix if it's the same for all profiles?

> Also this won't work for disa because there are two disa profiles:
> 
> * xccdf_org.ssgproject.content_profile_stig
> * xccdf_org.ssgproject.content_profile_stig_gui
> 
> And the command will actually select the gui one, which is not what we want.

This works fine, please be aware of the `:` after the `{{ he_openscap_profile_name }}`.

Reply:
> > Why user cannot simply specify the whole name instead of this filtering?
> 
> The user simply needs to specify `stig` or `pci-dss` which IMO is easier than specifying the entire name. Why do we need this prefix if it's the same for all profiles?
>
> > Also this won't work for disa because there are two disa profiles:
> > 
> > * xccdf_org.ssgproject.content_profile_stig
> > * xccdf_org.ssgproject.content_profile_stig_gui
> > 
> > And the command will actually select the gui one, which is not what we want.
> 
> This works fine, please be aware of the `:` after the `{{ he_openscap_profile_name }}`.

Ok did not notice that, makes sense then, thanks

Reply:
OK, let's hope we won't need any other profile in the near future.